test/batchedmul.jl

```diff
@@ -119,17 +119,21 @@ end
     end
 end
 
+perm_12(A) = PermutedDimsArray(A, (2,1,3))
```

Comment:
Can you move these inside the testset

Reply:
These don't depend on the T being looped over there. Seems clearer not to put them at the top of the file with all the other functions.